app/src/main/java/com/example/lhi06/residue/ReviewAdapter.java

```diff
@@ -59,10 +82,11 @@ public int getItemCount() {
 
     class ReviewViewHolder extends RecyclerView.ViewHolder {
 
-        @BindView(R.id.linearLayout_row) View rootView;
+        @BindView(R.id.review_preview_row) LinearLayout rootView;
```

Comment:
because i was having an issue where it couldnt find the layout and i found it easier to read as a LinearLayout, is this a bad change or?